bgpd/bgp_community.c

```diff
@@ -497,6 +497,9 @@ void community_unintern(struct community **com)
 {
 	struct community *ret;
 
+	if (!*com)
```

Comment:
why is this check removed ?

Reply:
it's added, not removed :)

Reply:
Sorry. Looks like part of my brain is swapped. :) 
Thanks.